dds/DCPS/GuidUtils.cpp

```diff
@@ -51,6 +52,13 @@ to_string(const GUID_t& guid)
     '.' + to_string(guid.entityId);
 }
 
+void intersect(const RepoIdSet& a, const RepoIdSet& b, RepoIdSet& result)
+{
+  result.clear();
+  intersect_sorted_ranges(a.cbegin(), a.end(), b.cbegin(), b.end(),
```

Comment:
I'm not sure about the use of `cbegin` here as there's no matching `cend`, but probably more relevant is that `cbegin` is not supported pre-C++11.